yarn.lock

```diff
@@ -3157,6 +3157,30 @@
   resolved "https://registry.yarnpkg.com/@spectrum-css/vars/-/vars-3.0.0-beta.2.tgz#f0b3a2db44aa57b1a82e47ab392c716a3056a157"
   integrity sha512-HpcRDUkSjKVWUi7+jf6zp33YszXs3qFljaaNVTVOf0m0mqjWWXHxgLrvYlFFlHp5ITbNXds5Cb7EgiXCKmVIpA==
 
+"@spectrum-web-components/base@^0.1.0", "@spectrum-web-components/base@^0.1.3":
```

Comment:
After we fix the version listed for `@spectrum-web-components/base` in the package.json files, there should no longer be changes to the lock here.

Reply:
Now that you've updated the `package.json` files, please run `yarn` locally and commit the new lock (which should mean there are no more changes to the lock...).